scala/private/rules/scala_test.bzl

```diff
@@ -0,0 +1,228 @@
+"""Rules for writing tests with ScalaTest"""
+
+load(
+    "@io_bazel_rules_scala//scala/private:common_attributes.bzl",
+    "common_attrs",
+    "implicit_deps",
+    "launcher_template",
+)
+load("@io_bazel_rules_scala//scala/private:common.bzl", "sanitize_string_for_usage")
+load("@io_bazel_rules_scala//scala/private:common_outputs.bzl", "common_outputs")
+load(
+    "@io_bazel_rules_scala//scala/private:coverage_replacements_provider.bzl",
+    _coverage_replacements_provider = "coverage_replacements_provider",
+)
+load(
+    "@io_bazel_rules_scala//scala/private:rule_impls.bzl",
+    "collect_jars_from_common_ctx",
+    "declare_executable",
+    "expand_location",
+    "first_non_empty",
+    "get_scalac_provider",
+    "get_unused_dependency_checker_mode",
+    "scala_binary_common",
+    "write_executable",
+    "write_java_wrapper",
+)
+
+def _scala_test_flags(ctx):
+    # output report test duration
+    flags = "-oD"
+    if ctx.attr.full_stacktraces:
+        flags += "F"
+    else:
+        flags += "S"
+    if not ctx.attr.colors:
+        flags += "W"
+    return flags
+
+def _scala_test_impl(ctx):
+    if len(ctx.attr.suites) != 0:
```

Comment:
we can probably remove this... this has been here for a very long time, and the migration is pretty simple, just remove `ctx.attr.suites`.

Maybe do that in a follow up.

Reply:
I went ahead and did this in https://github.com/bazelbuild/rules_scala/pull/815/commits/9a69e3823376f74a01c52b2542c65476b79f3071 to also try a CI rebuild.